File: ЛБ. 09/Text5/Text5.py
#Дана строка S и текстовый файл. Добавить строку S в конец файла.
txt = open('text55.txt', 'a')
print("Введите текст, который должен быть записан в конец файла.")
a = input()
txt.write(a)
print("Готово!")
txt.close()

File: ptaskbook/text5.pas
{Дана строка S и текстовый файл. Добавить строку S в конец файла}
var
f:text;
s:string;
begin
  assign(f, 'test5.txt');
  append(f);
  readln(s);
  writeln(f);
  write(f, s);
  close(f);
end.